package.json

```diff
@@ -55,6 +55,7 @@
   },
   "dependencies": {
     "esm": "^3.2.25",
+    "npm": "^7.16.0",
```

Comment:
If we go with this, let's remember to remove this change. See https://highfidelity.slack.com/archives/C0XU8ESJC/p1627948590001600